bin/pycbc_optimize_snr

```diff
@@ -120,21 +125,21 @@ parser.add_argument('--verbose', action='store_true')
 parser.add_argument('--params-file', required=True,
                     help='Location of the attributes file created by PyCBC '
                          'Live')
-parser.add_argument("--data-files", type=str, nargs='+',
+parser.add_argument('--data-files', type=str, nargs='+',
                     action=MultiDetOptionAction,
                     metavar='IFO:DATA_FILE',
-                    help="Locations of the overwhitened data files produced "
-                         "by PyCBC Live.")
-parser.add_argument("--psd-files", type=str, nargs='+',
+                    help='Locations of the overwhitened data files produced '
+                         'by PyCBC Live.")
+parser.add_argument('--psd-files', type=str, nargs='+',
                     action=MultiDetOptionAction,
                     metavar='IFO:PSD_FILE',
-                    help="Locations of the PSD files produced "
-                         "by PyCBC Live.")
-parser.add_argument("--approximant", required=True,
-                    help="Waveform approximant string.")
-parser.add_argument("--snr-threshold", default=4.0,
-                    help="If the SNR in ifo X is below this threshold do not "
-                         "consider it part of the coincidence.")
+                    help='Locations of the PSD files produced '
+                         'by PyCBC Live.")
+parser.add_argument('--approximant', required=True,
+                    help='Waveform approximant string.')
+parser.add_argument('--snr-threshold', default=4.0,
+                    help='If the SNR in ifo X is below this threshold do not '
+                         'consider it part of the coincidence. Not implemented')
```

Comment:
I forget what is going on here, but I vote for just dropping the option if it is not used. @spxiwh?

Reply:
If you do drop it, make sure `pycbc_live` does not use it!

Reply:
Looks like it doesn't.